spec/migrations/20170115140217_update_ems_in_miq_alert_status_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_migration
+
+describe UpdateEmsInMiqAlertStatus do
+  let(:miq_alert_status_stub) { migration_stub(:MiqAlertStatus) }
+
+  migration_context :up do
+    it 'it sets ems_id for vms' do
+      ext = FactoryGirl.create(:ext_management_system, :id => 99)
+      vm = FactoryGirl.create(:vm_cloud, :ext_management_system => ext)
```

Comment:
The actual migrations look great.
Thanks for fixing that stuff.

@Fryguy are we trying to avoid `FactoryGirl` in migration specs?